src/Module/ModulePortfolios.php

```diff
@@ -208,12 +235,156 @@ protected function parsePortfolio(Portfolio $objItem, bool $blnAddArchive = fals
         }
 
         // Parse the URL if we have a jumpTo configured
-        if ($objTarget = $objItem->getRelated('pid')->getRelated('jumpTo')) {
-            $objPageData = (new PageFinder())->findAssociatedForLanguage($objTarget, $GLOBALS['TL_LANGUAGE']);
-            $params = (Config::get('useAutoItem') ? '/' : '/items/').($objItem->slug ?: $objItem->id);
-            $objTemplate->jumpTo = $objPageData->getFrontendUrl($params);
-        }
+        $objTemplate->jumpTo = $objItem->getUrl();
 
         return $objTemplate->parse();
     }
+
+    /**
+     * Find items from remote
+     * 
+     * @var array config
+     * @var PortfolioFeed feed
+     *
+     * @return Collection
+     * 
+     * @throws \Exception
+     */
+    protected function findRemoteItems($config, $feed, $page, $limit): Collection
+    {
+        $ch = curl_init();
+        $params = $this->formatConfigForRemote($config, $feed);
+        $url = $feed->readFromRemoteUrl . '/api/portfolio/items/' . $page . '/' . $limit . '?' . $params;
+
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+
+        $request = curl_exec($ch);
+        curl_close($ch);
+        $data = json_decode($request, true);
+
+        // We need to format a Collection of Portfolio
+        $items = [];
+        foreach ($data as $item) {
+            $objModel = new Portfolio();
+            $objModel->pid = $feed->id;
+
+            foreach ($item as $c => $v) {
+                switch($c) {
+                    case 'category':
+                        // skip
+                    break;
+
+                    default:
+                        $objModel->{$c} = $v;
+                }
+            }
+
+            $items[] = $objModel;
+        }
+
+        $objCollection = new Collection($items, 'tl_wem_portfolio');
+
+        return $objCollection;
+    }
+
+    /**
+     * Count items from remote
+     * 
+     * @var array config
+     * @var PortfolioFeed feed
+     *
+     * @return Portfolio
+     * 
+     * @throws \Exception
+     */
+    protected function countRemoteItems($config, $feed): int
+    {
+        $ch = curl_init();
+        
+        $params = $this->formatConfigForRemote($config, $feed);
+        $url = $feed->readFromRemoteUrl . '/api/portfolio/count?' . $params;
+
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+
+        $request = curl_exec($ch);
+        curl_close($ch);
+        $data = json_decode($request, true);
+
+        return $data['items'];
```

Comment:
`return (int) $data['items'];`

Reply:
Fixed in afcdbb2b0f84ea2ea466b6a7b8b36a28f1965a27